windows/src/desktop/inst/keymandesktop.wxs

```diff
@@ -51,7 +51,7 @@
     </Upgrade>
 
     <Property Id='OLDINSTALLDIR'>
-      <RegistrySearch Id='oldinstalldir_search' Key='Software\Keyman\Keyman Desktop' Root='HKLM' Name='root path' Type='raw' />
+      <RegistrySearch Id='oldinstalldir_search' Key='Software\Keyman\Keyman for Windows' Root='HKLM' Name='root path' Type='raw' />
```

Comment:
Is this old directory in the registry? Won't those be `Software\Keyman\Keyman Desktop`?

Reply:
Good catch